peter-lang-verlag.csl

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded">
+  <info>
+    <title>Peter Lang Verlag</title>
+    <title-short>Peter Lang</title-short>
+    <id>http://www.zotero.org/styles/peter-lang-verlag</id>
+    <link href="http://www.zotero.org/styles/peter-lang-verlag" rel="self"/>
+    <author>
```

Comment:
> You can upload the document you've been sent within your PR and use that GitHub link :)

@sixtyfive, if you can share the publisher's requirements, see https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/ for instructions how to attach files to GitHub pull requests.